tutorials/hello/hello_client/src/app/plugins/hello_info/endpoint.ts

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable prefer-const */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable dot-notation */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/await-thenable */
+/* eslint-disable @typescript-eslint/no-floating-promises */
+/* eslint-disable import/no-cycle */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable no-console */
+/* eslint-disable spaced-comment */
+import {
+    BasePluginEndpoint,
+    PluginEndpointContext,
+    BasePlugin,
+    db as liskDB,
+    cryptography,
+    codec,
+} from 'lisk-sdk';
+import {
+    getEventHelloInfo,
+    getLastCounter,
+    getLastEventHeight,
+    setEventHelloInfo,
+    setLastCounter,
+    setLastEventHeight
+} from './db';
+import { chainEventSchema } from './schemas';
+
+
+export class Endpoint extends BasePluginEndpoint {
+    private _client!: BasePlugin['apiClient'];
+    private _db!: liskDB.Database;
+
+    public init(db: liskDB.Database, apiClient: BasePlugin['apiClient']) {
+        this._db = db;
+        this._client = apiClient;
+    }
+
+    // Returns all Sender Addresses, Hello Messages and Block Height of the block where the Hello Event was emitted.
+    public async getAddressList(_context: PluginEndpointContext): Promise<any[]> {
```

Comment:
```suggestion
    public async getMessageList(_context: PluginEndpointContext): Promise<any[]> {
```

`getAdressList ` is not so fitting name for the endpoint, because it only returns addresses that also have sent a Hello message. `getMessageList` is therefore more fitting imho.